multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/Messages.java

```diff
@@ -160,6 +160,7 @@
     public static final String DB_TRANSACTION_TIMEOUT = "Database transaction timeout: {0} seconds";
     public static final String SNAKEYAML_MAX_ALIASES_FOR_COLLECTIONS = "SnakeYaml max aliases for collections: {0}";
     public static final String SERVICE_HANDLING_MAX_PARALLEL_THREADS = "Service handling max parallel threads: {0}";
+    public static final String ABORTED_OPERATIONS_TTL_IN_SECONDS = "Aborted operations ttl in seconds: {0}";
```

Comment:
Replace `ttl` with `TTL` in the message for consistency with the message on line 108.

Reply:
Done